CI/update_docker.sh

```diff
@@ -1,10 +1,10 @@
 #!/bin/bash
 
-set -e
+set -ex
 
 ubuntu_versions="16.04 18.04"
 for ubuntu_version in ${ubuntu_versions}; do
   image_name="svalinn/dagmc-ci-ubuntu-${ubuntu_version}:latest"
-  docker build -t ${image_name} --build-arg UBUNTU_VERSION=${ubuntu_version} .
+  docker build -t ${image_name} --build-arg UBUNTU_VERSION=${ubuntu_version} -f CI/Dockerfile .
```

Comment:
Is this script now intended to be run from outside the `CI` directory?

Reply:
I though it would make more sense to run from the top directory